### PR TITLE
fix: menu gradient in content

### PR DESCRIPTION
**Pull request by diegolikescode**
Hello! Take a look, I've fixed the #922 problem.

The problem is that it renders 2x, in the first (I believe it renders the input first) ``endVisible.value`` is always false, so when you add the first element and beyond it remains false. This is why the second time the content loads, since it has ``options``, it loads the way it should.

I hope it was helpful! Maybe a better solution is to render only once, but this is a solution for the problem raised in the issue

### CODE REVIEW

**Comment by zhen0** [comment]
Thanks for the contribution.  This looks fixed in the deploy preview to me but I'll let @znicholasbrown or @pleek91 have final approval.

**Review by znicholasbrown — APPROVED** [review]
This is great, thank you @diegolikescode !

**Pull request merged**
